eml/api/api.js

```diff
@@ -2,7 +2,7 @@ import axios from 'axios';
 
 const timeoutInMs = 1200;
 
-const url = 'https://educado-backend-staging-x7rgvjso4a-ew.a.run.app/';
```

Comment:
we should keep this no?

Reply:
Not sure. We shouldn't run on the staging server in development. But it would be much better with a .env file with these kinds of variables